packages/segmentedControl/components/SegmentedControlButton.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from "react";
+import { cx } from "emotion";
+import shortid from "shortid";
+import { segmentedControlButton, segmentedControlButtonActive } from "../style";
+import { visuallyHidden, display } from "../../shared/styles/styleUtils";
+
+export interface SegmentedControlButtonProps {
+  /**
+   * Unique identifier used for the input button
+   */
+  id?: string;
+  /**
+   * Whether the button has been selected
+   */
+  isActive?: boolean;
+  /**
+   * The name passed to the input. Shared by all inputs in the segmented control
+   */
+  name?: string;
```

Comment:
If I am understanding correctly, all the button in a group have the same name?

Reply:
yes! and that happens to be the `id`-prop of the `SegmentedControl` component.

one noticable implication is that we can never have two buttons "active" at the same time. so this component can't be (ab)used as a multiselect.

Reply:
Yes, because it's a group of checkbox inputs